api/src/main/java/org/openmrs/OrderGroup.java

```diff
@@ -36,6 +36,7 @@
 	private OrderSet orderSet;
 	
 	private OrderGroup parentOrderGroup;
+	private Concept orderGroupReason;
```

Comment:
Can you put a line between these variables?

Reply:
i have fixed it

Reply:
i have fixed it